CMakeLists.txt

```diff
@@ -673,7 +690,7 @@ if(opencoarrays_aware_compiler)
   # Integration tests verifying the use of libcaf_mpi in applications
   add_caf_test(hello_multiverse 2 hello_multiverse)
   add_caf_test(coarray_burgers_pde 2 coarray_burgers_pde)
-  add_caf_test(co_heat 2 co_heat)
+#  add_caf_test(co_heat 2 co_heat)
```

Comment:
Why was this test (co_heat) disabled?

Reply:
Er, I don't know exactly. I am in the impression, that it was done by a merge of master?

Reply:
@vehre No, you did it in https://github.com/sourceryinstitute/OpenCoarrays/commit/7197bc7c086fe30ebd7079f3ac30d3389bda9de1

Reply:
I'll reenable it if there was no intentional reason.

Reply:
May be it was just waking up my testing and I just forgot to remove the # on commit.